### DISCUSSION TITLE
Update the docs

### DISCUSSION THREAD

**Discussion by katodude** [tagyoureit/nodejs-poolController#890] [Nixie]
In my latest rebuild spent a bunch of time reading the docs. There are some minor tweaks we should probably do. 

First there is no longer a need for a 24vdc power supply. The MegaBas can be powered by 24VAC. This allows you to power the entire box with just one power supply. 

Also we are recommending using a DIN rail box and the all those transformers look ugly and are not Din rail. This is the one that I am using. We might want to consider recommending that. 

https://us.rs-online.com/product/rs-pro/7975948/70654176/

Also we should do a little more documentation around setting up pH control. Especially mentioning that you need to switch the EZO from Uart to IC2. (That chewed up a whole bunch of hours).

**Comment by truvec** [comment]
The use case for 24vdc is that you can power a peristalic pump from it. 

I have both 24vdc and 24 vac.
24vac for valves (the one you linked to)
24vdc for Mega-Bas and Perstalic Pump

Still thinking about the best way to get 12vdc for the 3 relay which needs 12V to drive the relays. Looking at a separate PSU or DC-DC Converter (24v-12V). Seems cumbersome. I might drop the 3 relay and use the 8 relay to drive SSRs. 

I wish I was able to code at a better level to support the project but you're right the docs need some updating. I'm going to chip in there. 

Thanks

**Reply by rstrouse** [reply]
The best I was able to find when I built the first Nixie was a 20va 24vac transformer on a DIN rail.  This would not have been enough to properly power 2 valve movements at the same time.  The 70va is capable of at least 3 with a bit of power to spare on other components so for most applications it is probably a good choice.

As for SSRs It truly depends on what you are powering with it.  If you are powering items like a spa blowers, pumps, or electric heaters please use a contactor (3hp relay).  These are typically found in either 24vac or 24vdc coils which you can use the existing transformers.  Also, never power a 24vdc contactor coil with AC it will eventually melt down,  You can use a buck converter to get to 12 or ~24vdc if you want so long as it is sized appropriately.  

An equivalent SSR to handle the load and reduce current leakage through it will be very expensive although I haven't looked at them for a very long time.  There are no fast or frequent switching requirements for this application so a contactor will outlast all of us.  Be very careful to ensure you have the proper SSR paired with the proper load when using SSRs if you must.  Take all of that with a grain of salt because I am not an EE.  Bigger hardware brains will prevail as the best I can do is be a browser jockey for information that is often incomplete, conflicting, or flat wrong.

Looking forward to any help on the docs that we can get.

**Reply by truvec** [reply]
I think you are correct that for this use case, the contactors are preferred. 

I have used SSRs in the past and like that, they don't make noise. It's a pet peeve of mine. But since I'm the proud new owner of a Stenner 24vdc acid pump I should probably throw all noise concerns out the window.

**Reply by katodude** [reply]
What is the process to help on the documentation. Do you want me to just grab a section and edit it and send it back to you?

**Reply by rstrouse** [reply]
> I have used SSRs in the past and like that, they don't make noise. It's a pet peeve of mine. But since I'm the proud new owner of a Stenner 24vdc acid pump I should probably throw all noise concerns out the window.

It does sound a lot like a duck doing unspeakable things to a fish.

> What is the process to help on the documentation. Do you want me to just grab a section and edit it and send it back to you?

@tagyoureit will need to weigh in on this.  These are all written in markdown.  If you do edit a section I can plug it in for you though as a collaborator.

**Reply by tagyoureit** [reply]
Catching up on some old threads I missed.  The wiki is open for anyone to edit.  Happy to have you contribute.
